docs/MOM_parameter_doc.all

```diff
@@ -1255,7 +1255,7 @@ TIDAL_SAL_FROM_FILE = False     !   [Boolean] default = False
 USE_PREVIOUS_TIDES = False      !   [Boolean] default = False
                                 ! If true, use the SAL from the previous iteration of the tides to facilitate
                                 ! convergent iteration. This is only used if TIDES is true.
-TIDE_USE_SAL_SCALAR = True      !   [Boolean] default = True
+TIDE_USE_SAL_SCALAR = False     !   [Boolean] default = True
```

Comment:
```suggestion
TIDE_USE_SAL_SCALAR = True     !   [Boolean] default = True
```
Set to default for consistency with `MOM_input` (value is ignored since TIDES is false)

Reply:
Thanks for the suggestion. It has been fixed.